src/platform/forms-system/test/js/utilities/ccStringSearch.unit.spec.js

```diff
@@ -0,0 +1,187 @@
+import { expect } from 'chai';
+import { getDisabilityLabels } from 'applications/disability-benefits/all-claims/content/disabilityLabels';
+import {
+  exportForTesting,
+  fullStringSimilaritySearch,
+} from '../../../src/js/utilities/cc-string-matching';
+
+const testDisabilityLabels = Object.values(getDisabilityLabels());
+
+const {
+  THRESHOLD,
+  stripCommonWords,
+  lcsScoreByWordSum,
+  countOverlapWords,
+  substringSearchCountOverlap,
+  lcsSingleVsMulti,
+} = exportForTesting;
+
+describe('stripCommonWords', () => {
+  it('should remove common words from the middle of a string', () => {
+    const testString = 'pain in my left knee';
+    const expectedString = 'pain knee';
+    expect(stripCommonWords(testString)).to.eql(expectedString);
+  });
+
+  it('should remove common words from the ends of a string', () => {
+    const testString = 'my left knee hurts, bilateral';
+    const expectedString = 'knee hurts,';
+    expect(stripCommonWords(testString)).to.eql(expectedString);
+  });
+});
+
+describe('lcsScoreByWordSum', () => {
+  it('should return integer score for similarity', () => {
+    const inputString = 'right acl tear in knee';
+    const disabilityWord = 'ACL tear (anterior cruciate ligament tear), right';
+    const expectedScore = 8;
```

Comment:
I think defining score in `lcsScoreByWordSum()` might help for updating the thresholds / tweaking scores in the future